src/hotspot/share/runtime/lockStack.inline.hpp

```diff
@@ -216,7 +216,6 @@ inline bool LockStack::contains(oop o) const {
 }
 
 inline void LockStack::oops_do(OopClosure* cl) {
-  verify("pre-oops-do");
```

Comment:
It would be nice to have a comment stating that we don't perform the "pre-oops-do" check because this could is used by the GC to fix the oops.